forest/shared/src/cli/mod.rs

```diff
@@ -30,6 +30,82 @@ pub static FOREST_VERSION_STRING: Lazy<String> =
 /// CLI options
 #[derive(StructOpt, Debug)]
 pub struct CliOpts {
+    /// A TOML file containing relevant configurations
+    #[structopt(short, long)]
+    pub config: Option<String>,
+    /// Allow RPC to be active or not (default: true)
+    #[structopt(short, long)]
+    pub rpc: Option<bool>,
```

Comment:
Does `forest-cli` use the `rpc` flag?

Reply:
There is a test that executes `forest-cli --rpc true --token Azazello config dump`. I (perhaps incorrectly) assumed that the test was there for a reason.

Reply:
You need to understand what that test does and then update it if necessary.